### ISSUE TITLE
[Bug] Spleeter hangs endlessly on "Tears of a Clown"

### ISSUE THREAD

**Issue by beckerdo** [deezer/spleeter#49]
## Description

On this environment, Spleeter hangs endlessly on "Tears of a Clown" and other certain songs. It works for the demo example and a majority of songs, but has an endless loop on some.

The hang is similar whether using the spleeter-cpu or spleeter-gpu activation. The hang is similar whether using 2 stem or 4 stem output. The hang is similar whether using WAV or MP3 input format.

## Step to reproduce

git clone https://github.com/Deezer/spleeter
conda env create -f spleeter/conda/spleeter-cpu.yaml
conda env create -f spleeter/conda/spleeter-gpu.yaml

conda activate spleeter-cpu
or
conda activate spleeter-gpu

## Output

```bash
(spleeter-cpu) E:\git\people\deezer>spleeter separate -i spleeter/spleeter separate -i spleeter/SmokeyRobinsonTheMiracles-2004-Motown1s-17-TearsofaClown.m4a -p spleeter:2stems -o output
INFO:tensorflow:Using config: {'_model_dir': 'pretrained_models\\2stems', '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_steps': None, '_save_checkpoints_secs': 600, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.7
}
, '_keep_checkpoint_max': 5, '_keep_checkpoint_every_n_hours': 10000, '_log_step_count_steps': 100, '_train_distribute': None, '_device_fn': None, '_protocol': None, '_eval_distribute': None, '_experimental_distribute': None, '_experimental_max_worker_delay_secs': None, '_service': None, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x0000017870639240>, '_task_type': 'worker', '_task_id': 0, '_global_id_in_cluster': 0, '_master': '', '_evaluation_master': '', '_is_chief': True, '_num_ps_replicas': 0, '_num_worker_replicas': 1}
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From c:\programdata\miniconda3\envs\spleeter-cpu\lib\site-packages\spleeter\model\functions\unet.py:29: The name tf.keras.initializers.he_uniform is deprecated. Please use tf.compat.v1.keras.initializers.he_uniform instead.

INFO:tensorflow:Apply unet for vocals_spectrogram
INFO:tensorflow:Apply unet for accompaniment_spectrogram
INFO:tensorflow:Done calling model_fn.
WARNING:tensorflow:From c:\programdata\miniconda3\envs\spleeter-cpu\lib\site-packages\tensorflow\python\ops\array_ops.py:1354: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
INFO:tensorflow:Graph was finalized.
WARNING:tensorflow:From c:\programdata\miniconda3\envs\spleeter-cpu\lib\site-packages\tensorflow\python\training\saver.py:1276: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Restoring parameters from pretrained_models\2stems\model
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:The dtype of the watched tensor must be floating (e.g. tf.float32), got tf.string
WARNING:tensorflow:The dtype of the watched tensor must be floating (e.g. tf.float32), got tf.int32
WARNING:tensorflow:The dtype of the watched tensor must be floating (e.g. tf.float32), got tf.string
INFO:tensorflow:Loading audio b'spleeter/SmokeyRobinsonTheMiracles-2004-Motown1s-17-TearsofaClown.m4a' from 0.0 to 600.0
```

## Environment

|                   |                                 |
| ----------------- | ------------------------------- |
| OS                | Windows  10 64 bit |
| Installation type | Conda            |
| RAM available     | 32G                             |
| Hardware spec     | CPU is Intel i7-6700K @ 4 GHz / GPU is NVidea GeForce GTX 1070

## Additional context

The particular song I used was Smokey Robinson and the Miracles "Tears of a Clown" from the 2004 CD Motown 1s. The song has changing tempo and some flutes and oboes that might make it tough to separate.

**Comment by Faylixe** [comment]
Duplicate with #29 and #30. We deployed a new version of _Spleeter_ on PyPi that fix the issues (1.4.3). Please pull the repository again and recreate a fresh new conda environment. It will download the latest release of _Spleeter_ and it should work as expected.

**Comment by beckerdo** [comment]
Thank you. A pull and update of spleeter fixed this issue.
